src/icons/fileIcons.ts

```diff
@@ -802,5 +802,11 @@ export const fileIcons: FileIcons = {
         { name: 'vim', fileExtensions: ['vimrc', 'gvimrc', 'exrc'] },
         { name: 'nest', fileNames: ['nest-cli.json', '.nest-cli.json', 'nestconfig.json', '.nestconfig.json'] },
         { name: 'moonscript', fileExtensions: ['moon'] },
+        {
+            name: 'uml',
+            // https://marketplace.visualstudio.com/items?itemName=jebbs.plantuml#user-content-supported-formats
```

Comment:
Could you please remove the comment here?

Reply:
Sure!

I'm curious, why? I thought it would be helpful for anyone wondering where the extensions came from.

Reply:
I understand your intention, but it's not the right place for me to add this information. If someone is really interested in where the extensions came from he/she can use "git blame" or search through the issues in GitHub. I want to avoid that data has to be transferred and stored which is only meta information and does not improve the functionality of the extension. Thanks for removing it 😉

Reply:
Got it.